emcc.py

```diff
@@ -1886,21 +1900,23 @@ def do_minify(): # minifies the code. this is also when we do certain optimizati
            '--mapFileBaseName', target,
            '--offset', str(0)])
 
-    # Move final output to the js target
-    shutil.move(final, js_target)
```

Comment:
No, the intent was not to save a move, but a copy.

Anyways, updated the pull request to change the line ending conversion always operate in place and never move or copy. Probably not that different in the end. How does that look now?